packages/react-core/src/components/Card/examples/Card.md

```diff
@@ -26,7 +27,7 @@ import PlusIcon from '@patternfly/react-icons/dist/esm/icons/plus-icon';
 
 ### Basic cards
 
-Basic cards typically have a `<CardTitle>`, `<CardBody>` and `<CardFooter>`. You may omit these components as needed, but it is recommended to at least include a `<CardBody>` to provide details about the card item.
+Basic cards typically have a `<CardTitle>`, `<CardSubtitle>`, `<CardBody>` and `<CardFooter>`. You may omit these components as needed, but it is recommended to at least include a `<CardBody>` to provide details about the card item.
```

Comment:
Would we want to say that cards will typically have a subtitle, or should the subtitles be just somsthing that can be added but aren't usually there?

Reply:
this has been removed and we have a seperate demo